pkg/config/config.go

```diff
@@ -132,6 +132,12 @@ func (c *Config) InitConfig() {
 		}).Debug("Using profiles file")
 	}
 
+	if os.Getenv("STRIPE_CLI_CANARY") == "true" {
```

Comment:
i'm peaceful with it for now since it's meant as a temporary patch to vet the fix, but i'd feel a lot more comfortable if this was in a helper function since the result typing is loose and we're using a bool-as-a-string from an env var, which is easy to typo

Reply:
Makes sense! I need to resolve this to be able to merge but will use a helper method the next time we need an env var check.